pages/dkp/kommander/2.2/insights/insights-view/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: layout.pug
+navigationTitle: View insight items
```

Comment:
Similar comment as above,  should this be "Insight Items"?   Or "Insights items"?  I.e is it always plural, and what is the capitalization of each?

I think `Insights` should be capitalized, at least.    We should be consistent across these pages on this.